test/Microsoft.AspNetCore.SignalR.Common.Tests/Internal/Protocol/CustomObject.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.SignalR.Common.Tests.Internal.Protocol
+{
+    public class CustomObject : IEquatable<CustomObject>
```

Comment:
Now that this is separate from the test file, can we call it `TestCustomObject` or something?